src/data/types.rs

```diff
@@ -491,6 +491,7 @@ pub(crate) mod tests {
         assert_decl_display(
             "int *(*jynelson)(int (*)(int));",
             "int *(*jynelson)(int (*)(int));",
-        )
+        );
+        assert_decl_display("int f(...);", "int f(...);");
```

Comment:
This should be caught as an error, so it will need to be removed when I fix that bug. It's fine for now though.

Reply:
Opened https://github.com/jyn514/rcc/issues/410